app/controllers/summary_controller.rb

```diff
@@ -2,27 +2,38 @@ class SummaryController < ApplicationController
   before_action :authorize
 
   def index
-    client_namespace = current_user.client_slug.titleize
-    @client_summarizer = ClientSummarizer.new(
-      client_namespace: client_namespace,
-      fiscal_year: params[:fiscal_year]
-    )
-    @summary = @client_summarizer.run
+    client_namespaces = if current_user.gateway_admin?
```

Comment:
1. The card calls for gateway admins to have that power, but now that you mention it, it makes sense for regular admins (but not client admins) to have it too. I'll tweak it.

2. OK!